src/containerapp/azext_containerapp/_help.py

```diff
@@ -0,0 +1,246 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
@anthonychu please review help file - this will be documentation that will published publicly.